CHANGELOG.md

```diff
@@ -9,7 +9,10 @@ Monitor the release status by regions at [AKS-Release-Tracker](http://aka.ms/aks
 * Starting with Kubernetes 1.25, the following changes will be made default:
   * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
   * Windows Server 2022 will be the default Windows host. Important, old windows 2019 containers will not work on windows server 2022 hosts.
+<<<<<<< HEAD
```

Comment:
these need to be removed

Reply:
I'm not sure why this is showing up here. Must have been something that happened with the rebase because those merge conflict marks were not showing up on my local copy.